storagenode/monitor/monitor_test.go

```diff
@@ -20,17 +20,12 @@ func TestMonitor(t *testing.T) {
 	testplanet.Run(t, testplanet.Config{
 		SatelliteCount: 1, StorageNodeCount: 6, UplinkCount: 1,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
-		satellite := planet.Satellites[0]
-
-		var freeBandwidth int64
+		var freeBandwidthInit int64
 		for _, storageNode := range planet.StorageNodes {
 			storageNode.Storage2.Monitor.Loop.Pause()
 
-			info, err := satellite.Contact.Service.FetchInfo(ctx, storageNode.Local().Node)
```

Comment:
This was specifically testing also `FetchInfo`, it still needs to ensure that the method works.

Reply:
We deleted fetch info

Reply:
https://github.com/storj/storj/pull/3175/files/c9cff85eac90f806b691f9d75118f9624ed9891d?file-filters%5B%5D=.go&file-filters%5B%5D=.lock&file-filters%5B%5D=.proto&hide-deleted-files=true#diff-555a3b020110a77ab5fc3e7fde16a5f5L70